src/ctapipe/image/muon/fitting.py

```diff
@@ -12,7 +13,7 @@
     Minuit = None
 
 
-def kundu_chaudhuri_circle_fit(x, y, weights):
+def kundu_chaudhuri_circle_fit(x, y, weights, dummy_errors_flag=True):
```

Comment:
I think the default should be False (correct error calculation)
```suggestion
def kundu_chaudhuri_circle_fit(x, y, weights, dummy_errors_flag=False):
```